### PR TITLE
NAS-107574 / 21.02 / NAS-107574

### PR DESCRIPTION
**Pull request by ryousaf**

### CODE REVIEW

**Review by dszidi — CHANGES_REQUESTED** [review]
The zvol field should be populated with the newly create volume once the wizard is submitted. Currently it only shows "New"

File: src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts [hunk 1]
```diff
@@ -1064,6 +1064,7 @@ export class VolumesListTableConfig implements InputTableConf {
           name: T('Add Zvol'),
           label: T("Add Zvol"),
           onClick: (row1) => {
+            console.log('addZvol', rowData.id);
```
Comment (william-gr) [inline src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts:1067]: Leftover debug?

**Review by william-gr — COMMENTED** [review]
@ryousaf would you care to post a few screenshots so we can see what it looks like?

File: src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts (hunk 1 above)
Comment (ryousaf) [reply src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts:1067]: Yes. After the review by Damian, I was making further changes. Will clean it up.

**Comment by ryousaf** [comment]
> @ryousaf would you care to post a few screenshots so we can see what it looks like?

Here are some screenshots as you go through the process of adding a new Zvol while adding a new disk.

Screenshot 2020-12-23 201318
Screenshot 2020-12-23 201337
Screenshot 2020-12-23 201347
Screenshot 2020-12-23 201402
Screenshot 2020-12-23 201438
Screenshot 2020-12-23 201451
Screenshot 2020-12-23 201500
Screenshot 2020-12-23 201511
Screenshot 2020-12-23 201522
Screenshot 2020-12-23 201719
Screenshot 2020-12-23 201740
Screenshot 2020-12-23 201752

**Comment by william-gr** [comment]
Thanks!

A few notes:
- The padding is looking very odd for the Device Add Form.
- Zvol Directory isn't really a thing. I wonder if we have a selector that let us choose a ZFS Filesystem, no path. Just `tank/foo`, instead of `/mnt/tank/foo`
- The Add New option, how hard is it to keep it sticky at the end of the list? and the let scroll work over the existing items?

**Comment by dszidi** [comment]
Adding the class `mat-card-padded` to `<mat-card>` in the device-add.component.html file will fix your padding issue

**Review by dszidi — APPROVED** [review]
LGTM

**Pull request merged**